finalPrep/modZeroHero.js

```diff
@@ -1,52 +1,79 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
+var heroName = "Batman";
+var specialAbility = "just being rich";
 
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
+var greeeting = ("I'm " + heroName + "!");
+var catchphrase = (`I'll save Gotham by ${specialAbility}.`);
+
 
 // Declare two variables - power AND energy - set to integers
+var power = 45;
+var energy = 80;
 
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
-
+var fullPower = (power * 500);
+var fullEnergy = (power + 150);
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var isHuman = true;
+var identityConcealed = true;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
+var archEnemies = ["The Joker", "Poison Ivy", "Dr. Freeze"];
+var sidekicks = ["Robin", "CatWoman", "Alfred"];
 
 // Print the first sidekick to your console
-
+console.log(sidekicks[0]);
 // Print the last archEnemy to the console
-
+console.log(archEnemies[2]);
```

Comment:
Nice job with your bracket notation. A next level is to think about how to print the last element regardless of how long that array is.